[2024-05-01T13:56:15.396+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-01T13:56:15.429+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: try_dag.model_2 manual__2024-05-01T13:56:11.620050+00:00 [queued]>
[2024-05-01T13:56:15.439+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: try_dag.model_2 manual__2024-05-01T13:56:11.620050+00:00 [queued]>
[2024-05-01T13:56:15.440+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-01T13:56:15.492+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): model_2> on 2024-05-01 13:56:11.620050+00:00
[2024-05-01T13:56:15.502+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=114) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-01T13:56:15.504+0000] {standard_task_runner.py:63} INFO - Started process 118 to run task
[2024-05-01T13:56:15.509+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'try_dag', 'model_2', 'manual__2024-05-01T13:56:11.620050+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/sample_dag.py', '--cfg-path', '/tmp/tmpvgcmhhh8']
[2024-05-01T13:56:15.512+0000] {standard_task_runner.py:91} INFO - Job 37: Subtask model_2
[2024-05-01T13:56:15.593+0000] {task_command.py:426} INFO - Running <TaskInstance: try_dag.model_2 manual__2024-05-01T13:56:11.620050+00:00 [running]> on host cab22781d943
[2024-05-01T13:56:15.762+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='try_dag' AIRFLOW_CTX_TASK_ID='model_2' AIRFLOW_CTX_EXECUTION_DATE='2024-05-01T13:56:11.620050+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-01T13:56:11.620050+00:00'
[2024-05-01T13:56:15.764+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-01T13:56:15.765+0000] {python.py:237} INFO - Done. Returned value was: 3.1622776601683795
[2024-05-01T13:56:15.766+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-01T13:56:15.817+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=try_dag, task_id=model_2, execution_date=20240501T135611, start_date=20240501T135615, end_date=20240501T135615
[2024-05-01T13:56:15.884+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-01T13:56:15.909+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-01T13:56:15.950+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
